examples/ui-demo/src/hooks/useMAv2Client.ts

```diff
@@ -22,15 +21,23 @@ type Client = ModularAccountV2Client<
     AlchemyWebSigner | LocalAccountSigner<PrivateKeyAccount>
   >;
 
-export const useSma7702Client = (
-  localKeyOverride?:
-    | {
-        readonly key: Hex;
-        readonly entityId: number;
-        readonly accountAddress?: Address;
-      }
-    | undefined
-): {
+// Hook that creates an MAv2 client that can be used for things that
+// @account-kit/react doesn't yet support, such as session keys.
+export const useMAv2Client = ({
```

Comment:
This is used for:
- 7702 client for minting (since switching modes on the standard hooks is still WIP)
- 7702 and default client for session keys (since standard hooks don't have support yet)

Reply:
Are we using "MAv2" or "MA" anywhere else? Nit: would prefer to spell this out `useModularAccountV2Client` for clarity if not.

Reply:
nope, just renamed it 👍